src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -48,11 +48,11 @@
   -->
 
   <xsl:template name="pickfont-sans">
-    <xsl:text>Arial,sans-serif</xsl:text>
+    <xsl:text>Liberation Sans,Arial,sans-serif</xsl:text>
   </xsl:template>
 
   <xsl:template name="pickfont-serif">
-    <xsl:text>Georgia,serif</xsl:text>
+    <xsl:text>DejaVu Serif,Liberation Serif,serif</xsl:text>
```

Comment:
Actually, so that we don't throw off existing users, I think we should keep Arial as the first choice for sans and Georgia as the first choice for serif.